docs: Documentation for MetricPipeline (#362)
kyma-project/telemetry-manager: docs/user/03-traces.md

![Distributed tracing](./assets/traces-intro.drawio.svg)

> The goal of the Telemetry Module is to support you in collecting all relevant trace data in a Kyma cluster, enrich them and ship them to a backend for further analysis. Kyma modules like Istio or Serverless contribute traces transparently. Multiple [vendors for OTLP-based backends](https://opentelemetry.io/ecosystem/vendors/) are available.

Reviewer: ```suggestion
The goal of the Telemetry Module is to support you in collecting all relevant trace data in a Kyma cluster, enrich them and ship them to a backend for further analysis. Kyma modules like Istio or Serverless contribute traces transparently. You can choose among multiple [vendors for OTLP-based backends](https://opentelemetry.io/ecosystem/vendors/).
```

  name: backend
  namespace: default
stringData:
  endpoint: https://backend.example.com:4317
  user: myUser
  password: XXX
  token: Bearer YYY
```

### Step 4: Rotate the Secret

Telemetry Manager continuously watches the Secret referenced with the **secretKeyRef** construct. You can update the Secret’s values, and Telemetry Manager detects the changes and applies the new Secret to the setup.
If you use a Secret owned by the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator), you can configure an automated rotation using a `credentialsRotationPolicy` with a specific `rotationFrequency` and don’t have to intervene manually.

### Result

> To activate and verify the constructed TracePipeline, follow these steps:

Reviewer: same as above, why is activating the pipeline not part of the procedure? How common is it that users construct the pipeline but won't activate it?

## Troubleshooting

- Symptom: No traces are arriving at the destination.

>    Cause: The backend is not reachable or wrong authentication credentials are used

Reviewer: ```suggestion
   Cause: The backend is not reachable or wrong authentication credentials are used.
```
